feature-updater/publish.sh

```diff
@@ -0,0 +1,58 @@
+ #!/bin/bash
+echo "\nCreating 'Feature Update' for BPS Tooling Feature\n"
```

Comment:
Fixed in https://github.com/wso2/devstudio-tooling-bps/commit/3f440647b4d817de41875b600c296961b2662bc1